### DISCUSSION TITLE
Notification Component

### DISCUSSION THREAD

**Discussion by khlystou** [moonshine-software/moonshine#998] [Ideas]
Сделать изолированный компонент нотификации для того, что бы можно было использовать его не только в topLayout но и в любом другом компоненте, будь то Header, TopBar и тд. 
Сделать полинг для этого компонента. Сейчас, если страница не перезагружена, то и не видно уведомления, что не очень удобно.

**Comment by lee-to** [comment]
Перенес в issue
